docs/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to stake on EON
+---
+
+# How to stake on EON
+A forger is required to run a node; if you still don't have a node, here are some [instructions](../04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md).  
+The steps below require some basic general knowledge; a web application for staking is under construction and will be made available soon.
+
+#### Create Forger keys
+On your local node, create new forger keys by making a `POST` request to `/wallet/createVrfSecret`.
+The request will return your `publicKey` (later called `vrfPubKey`), while the privateKey will be stored in the node's wallet.
+
+```
+{
+  "result" : {
+    "proposition" : {
+      "publicKey" : "4439cbfd50af1b846e5ef06889d3192ef7a459bdd4640dc6da506062de43113c80"
+    }
+  }
+}
+``` 
+
+At this point there are two ways of staking on EON: via HTTP endpoint or via smart contract.
+#### HTTP endpoint
+Make a `POST` request to `/transaction/makeForgerStake` with body
+```
+{
+    "forgerStakeInfo": {
+        "ownerAddress": "e8674062598b86d8258257a4d79e99d15594d864", 
+        "blockSignPublicKey": "e8674062598b86d8258257a4d79e99d15594d864", 
+        "vrfPubKey": "4439cbfd50af1b846e5ef06889d3192ef7a459bdd4640dc6da506062de43113c80", 
+        "value": 10000
+    }
+}
+```
+where:
+- `ownerAddress`: is the address that is requesting to stake such amount and the only one allowed to unstake (withdraw) the specified amount;
+- `blockSignPublicKey`: is the address that will sign the block when forged. It can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used;
+- `vrfPubKey`: this is the key created in the previous step;
+- `value`: the amount of ZEN expressed in Satoshis (10^8) to put on stake, that will represent your voting power.
+
+#### How to create a forging stake using web3js
+Below there is an example (written in javascript to run in Remix IDE) of how to create the forging stake by interacting with the contract:
```

Comment:
change to --> Here is an example (written in javascript running in Remix IDE) on creating the forging stake with smart contract interaction: